multidoc/make.jl

```diff
@@ -56,8 +80,9 @@ MultiDocumenter.make(
 
 # Deploy to Github with running as a Github action
 if haskey(ENV, "GITHUB_ACTIONS")
-    isdir("multidoc-preview") || mkdir("multidoc-preview")
-    multidocroot = normpath(joinpath(@__DIR__, "../multidoc-preview/"))
+    output_dir = "library"
+    isdir(output_dir) || mkdir(output_dir)
```

Comment:
This can be simplified to
```suggestion
    mkpath(output_dir)
```